### ISSUE TITLE
inquiry

### ISSUE THREAD

**Issue by ryanseee** [ryansee/cepfinalproj2019#1]
hey, i'd like to inquire about your github username that you are using. really hope you would get back to me as it is urgent.

**Comment by ryansee** [comment]
hi, my github username is my real name. May I know why you are asking?

**Comment by ryanseee** [comment]
i see that you werent very active on github, so i wanted to ask if it was possible if you could let me have your github name @ryansee? I'd be starting a project, quite a significant one and would like to have this name to identify myself. hope you could help me out.

**Comment by ryansee** [comment]
hello, sorry I think I might be using github more often after I graduate from high school, hope you can understand. Maybe u could add some numbers behind your username? For example, ryansee123? That way people will still be to know your real name. Once again, I'm very sorry, hope you can forgive me :'(

**Comment by ryanseee** [comment]
hey, do you have an email or on any communication platform I could connect you with?

**Comment by ryansee** [comment]
Hello! Yes, is there something you want to tell me?

**Comment by ryanseee** [comment]
yeah, thought if by any chance you'd ever release the username you could let me know haha.

**Comment by ryansee** [comment]
ah i see, you could leave your email i guess
